modules/plugin/jdbc/jdbc-postgis/src/test/java/org/geotools/data/postgis/PostGISJsonOnlineTest.java

```diff
@@ -146,6 +146,22 @@ public void testJSONPointerFunction() throws Exception {
             }
             assertEquals(1, size);
         }
+        if (pgJsonTestSetup.supportJsonB) {
```

Comment:
All of these seem basically copy/paste with the name of the column changed. Could you factor out a method that takes the name of the column instead?

Reply:
done